ethcore/src/miner/miner.rs

```diff
@@ -876,6 +877,13 @@ impl Miner {
 			SealingState::NotReady => { self.maybe_enable_sealing(); },
 		}
 	}
+
+	/// Call `update_sealing` if needed
+	pub fn maybe_update_sealing<C: miner::BlockChainClient>(&self, chain: &C) {
+		if self.transaction_queue.has_local_pending_transactions() {
```

Comment:
I'm not sure about this. The implementation is very specific for `NullEngine` (i.e. checks local transactions), while we have plenty of other conditions that affect when to seal. Why can't we just call `update_sealing` instead?

Reply:
I think this is still valid.

Reply:
Ah sorry, I thought you were talking about the next line, where I first used `prepare_and_update_sealing`. I just updated it.

Reply:
I guess then there is no point in adding `maybe_update_sealing` to the traits and just call `update_sealing` directly?